Compilation error

RigelEngine/src/assets/user_profile_import.hpp:31:25: error: ‘uint8_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
   31 |   using ScanCode = std::uint8_t;
      |                         ^~~~~~~
      |                         wint_t
RigelEngine/src/assets/user_profile_import.hpp:33:3: error: ‘ScanCode’ no nombra a un tipo
   33 |   ScanCode mUpKeybinding;
      |   ^~~~~~~~
RigelEngine/src/assets/user_profile_import.hpp:34:3: error: ‘ScanCode’ no nombra a un tipo
   34 |   ScanCode mDownKeybinding;
      |   ^~~~~~~~
RigelEngine/src/assets/user_profile_import.hpp:35:3: error: ‘ScanCode’ no nombra a un tipo
   35 |   ScanCode mLeftKeybinding;
      |   ^~~~~~~~
RigelEngine/src/assets/user_profile_import.hpp:36:3: error: ‘ScanCode’ no nombra a un tipo
   36 |   ScanCode mRightKeybinding;
      |   ^~~~~~~~
RigelEngine/src/assets/user_profile_import.hpp:37:3: error: ‘ScanCode’ no nombra a un tipo
   37 |   ScanCode mJumpKeybinding;
      |   ^~~~~~~~
RigelEngine/src/assets/user_profile_import.hpp:38:3: error: ‘ScanCode’ no nombra a un tipo
   38 |   ScanCode mFireKeybinding;
      |   ^~~~~~~~
RigelEngine/src/assets/user_profile_import.hpp:47:8: error: ‘uint8_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?
   47 |   std::uint8_t mGameSpeedIndex;
      |        ^~~~~~~
      |        wint_t
[ 24%] Building CXX object src/CMakeFiles/rigel_core.dir/audio/sound_system.cpp.o
RigelEngine/src/assets/user_profile_import.cpp: En la función lambda:
RigelEngine/src/assets/user_profile_import.cpp:194:37: error: ‘ScanCode’ en ‘struct rigel::assets::GameOptions’ no nombra un tipo
  194 |     return static_cast<GameOptions::ScanCode>(
      |                                     ^~~~~~~~
RigelEngine/src/assets/user_profile_import.cpp: In function ‘std::optional<rigel::assets::GameOptions> rigel::assets::loadOptions(const std::string&)’:
RigelEngine/src/assets/user_profile_import.cpp:206:12: error: ‘struct rigel::assets::GameOptions’ has no member named ‘mUpKeybinding’
  206 |     result.mUpKeybinding = asScanCode(reader.readU16());
      |            ^~~~~~~~~~~~~
RigelEngine/src/assets/user_profile_import.cpp:207:12: error: ‘struct rigel::assets::GameOptions’ has no member named ‘mDownKeybinding’
  207 |     result.mDownKeybinding = asScanCode(reader.readU16());
      |            ^~~~~~~~~~~~~~~
RigelEngine/src/assets/user_profile_import.cpp:208:12: error: ‘struct rigel::assets::GameOptions’ has no member named ‘mLeftKeybinding’
  208 |     result.mLeftKeybinding = asScanCode(reader.readU16());
      |            ^~~~~~~~~~~~~~~
RigelEngine/src/assets/user_profile_import.cpp:209:12: error: ‘struct rigel::assets::GameOptions’ has no member named ‘mRightKeybinding’
  209 |     result.mRightKeybinding = asScanCode(reader.readU16());
      |            ^~~~~~~~~~~~~~~~
RigelEngine/src/assets/user_profile_import.cpp:210:12: error: ‘struct rigel::assets::GameOptions’ has no member named ‘mJumpKeybinding’
  210 |     result.mJumpKeybinding = asScanCode(reader.readU16());
      |            ^~~~~~~~~~~~~~~
RigelEngine/src/assets/user_profile_import.cpp:211:12: error: ‘struct rigel::assets::GameOptions’ has no member named ‘mFireKeybinding’
  211 |     result.mFireKeybinding = asScanCode(reader.readU16());
      |            ^~~~~~~~~~~~~~~
RigelEngine/src/assets/user_profile_import.cpp:223:12: error: ‘struct rigel::assets::GameOptions’ has no member named ‘mGameSpeedIndex’
  223 |     result.mGameSpeedIndex =
      |            ^~~~~~~~~~~~~~~
[ 24%] Building CXX object src/CMakeFiles/rigel_core.dir/base/array_view.cpp.o
[ 25%] Building CXX object src/CMakeFiles/rigel_core.dir/base/image.cpp.o
make[2]: *** [src/CMakeFiles/rigel_core.dir/build.make:244: src/CMakeFiles/rigel_core.dir/assets/user_profile_import.cpp.o] Error 1
make[2]: *** Se espera a que terminen otras tareas....
make[1]: *** [CMakeFiles/Makefile2:506: src/CMakeFiles/rigel_core.dir/all] Error 2
make: *** [Makefile:156: all] Error 2

--- a/.github/workflows/bundling.yml
+++ b/.github/workflows/bundling.yml
@@ -30,24 +30,26 @@ jobs:
       with:
         name: ubuntu_deb_build
         path: build/packages/*.deb*
-  build_package_osx:
-    runs-on: macos-11
-    steps:
-    - uses: actions/checkout@v2
-      with:
-        submodules: recursive
-    - name: Install dependencies
-      run: brew install sdl2 sdl2_mixer
-    - name: Run CMake
-      run: cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DCMAKE_OSX_DEPLOYMENT_TARGET=10.15
-    - name: Build
-      run: cd build && make -j2
-    - name: Package (dmg)
-      run: cd build && cpack -G "DragNDrop" .. && mv packages/rigelengine.dmg packages/rigelengine_mac_x64.dmg && mv packages/rigelengine.dmg.sha256 packages/rigelengine_mac_x64.dmg.sha256
-    - uses: actions/upload-artifact@v2
-      with:
-        name: macos_dmg_build
-        path: build/packages/*.dmg*
+#  build_package_osx:
+#    runs-on: macos-11
+#    steps:
+#    - uses: actions/checkout@v2
+#      with:
+#        submodules: recursive
+#    - name: Install dependencies
+#      run: brew install sdl2 sdl2_mixer
+#    - name: Run CMake
+#      run: cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DCMAKE_OSX_DEPLOYMENT_TARGET=10.15
+#    - name: Build
+#      run: cd build && make -j2
+#    - name: Package (dmg)
+#      run: cd build && cpack -G "DragNDrop" .. && mv packages/rigelengine.dmg packages/rigelengine_mac_x64.dmg && mv packages/rigelengine.dmg.sha256 packages/rigelengine_mac_x64.dmg.sha256
+#    - uses: actions/upload-artifact@v2
+#      with:
+#        name: macos_dmg_build
+#        path: build/packages/*.dmg*
+#
+# Older approach, has been broken for a while:
 #  build_package_osx:
 #    runs-on: macos-11
 #    steps:

--- a/src/assets/user_profile_import.hpp
+++ b/src/assets/user_profile_import.hpp
@@ -20,6 +20,7 @@
 #include "data/saved_game.hpp"
 
 #include <array>
+#include <cstdint>
 #include <optional>
 
 

